package.json

```diff
@@ -50,6 +52,7 @@
     "babel-plugin-macros": "2.8.0",
     "babel-plugin-remove-export-keywords": "^1.6.2",
     "babel-plugin-transform-vue-jsx": "4.0.1",
+    "dtslint": "3.6.3",
```

Comment:
pulled dtslint to the top level, since the same version can and should be used to test all packages